rln/src/public.rs

```diff
@@ -153,36 +139,33 @@ impl RLN {
     /// let resources_folder = "./resources/tree_height_20/";
     ///
     /// let mut resources: Vec<Vec<u8>> = Vec::new();
-    /// for filename in ["rln.wasm", "rln_final.zkey", "verification_key.arkvkey"] {
+    /// for filename in ["rln_final.zkey", "verification_key.arkvkey"] {
     ///     let fullpath = format!("{resources_folder}{filename}");
     ///     let mut file = File::open(&fullpath).expect("no file found");
     ///     let metadata = std::fs::metadata(&fullpath).expect("unable to read metadata");
     ///     let mut buffer = vec![0; metadata.len() as usize];
     ///     file.read_exact(&mut buffer).expect("buffer overflow");
     ///     resources.push(buffer);
-    ///     let tree_config = "{}".to_string();
-    ///     let tree_config_input = &Buffer::from(tree_config.as_bytes());
     /// }
     ///
+    /// let tree_config = "{}".to_string();
```

Comment:
Are these lines needed?

Reply:
I think so. In the previous version these lines were within the loop body and therefore "tree_config" was not accessible by "new_with_params".

Reply:
btw, it's wrong config definition. It should be empty string 
```
let tree_config = "".to_string();
```

Reply:
Are you sure? The previous version of the code contained "let tree_config = "{}".to_string();" within the loop body.